userspace/libscap/engine/savefile/converter/converter.cpp

```diff
@@ -618,7 +619,19 @@ int push_parameter_from_callback(scap_evt *new_evt,
 	const auto len_size = get_param_len_size(new_evt);
 	const auto min_param_len = get_min_param_len_from_type(param_type);
 	const auto max_param_len = get_max_param_len_from_type(param_type, len_size);
-	const auto buffer = callback(scap_evt_param_reader{*old_evt}, min_param_len, max_param_len);
+	std::vector<char> buffer;
+	try {
+		buffer = callback(scap_evt_param_reader{*old_evt}, min_param_len, max_param_len);
+	} catch(...) {
+		// todo: log info about exception.
```

Comment:
This will require the converter to log something, but currently it has no access to any logging framework.